R/LSgameofchance.R

```diff
@@ -59,15 +60,15 @@ compare_function1 <- function(p,m1,n1,t,simulation){
   record_game <- vector()
   for (i in 1:simulation){
     # copy m and n
-    record_trial <- rbind(m,n)
+    record_trial <- rbind(m, n)
     # keep running the game until one of them wins
-    while (record_trial[1,]*record_trial[2,]!= 0){
-      record_trial <- record_trial - rmultinom(1,1,c(p,1-p)) #updating the recorded result 
+    while (record_trial[1, ] * record_trial[2, ] != 0){
```

Comment:
the JASP style requires a space when accessing matrices/data.frames/etc after space